README.md

```diff
@@ -27,6 +27,18 @@ to locally, then run:
 git clone https://github.com/openT2T/console.git
 ```
 
+## WINK configuration:
+## Create winkConfig.ts file under "app\pages\onboarding\winkOnboardingPage" with the following contents.

```

Comment:
I don't think these two lines (31 and 32) need the ## prefix since only the ## Wink Configuration: line (30) is the subheading.